sample-apps/donet-sample-app/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+services:
+  otel:
+    image: amazon/aws-otel-collector:latest
+    command: --config /config/collector-config-local.yml
+    volumes:
+      - ~/.aws:/root/.aws:ro
+      - .:/config
```

Comment:
Is this copying everything into the `/config` directory of the container? Is there a reason we want to do this?